src/main/java/com/salesforce/dynamodbv2/mt/repo/MtDynamoDbTableDescriptionRepo.java

```diff
@@ -139,41 +135,48 @@ private TableDescription getTableDescriptionFromCache(String tableName) throws R
         }
     }
 
-    private TableDescription getTableDescriptionNoCache(String tableName) {
+    private MtTableDescription getTableDescriptionNoCache(String tableName) {
```

Comment:
This is where we first try to get a description for the given virtual table name at the current context, and if it doesn't exist, see if there's virtual multitenant table with this name at the top-level context.